opentracing/event_receiver.go

```diff
@@ -0,0 +1,35 @@
+package opentracing
+
+import (
+	"context"
+
+	ot "github.com/opentracing/opentracing-go"
+	otext "github.com/opentracing/opentracing-go/ext"
+	otlog "github.com/opentracing/opentracing-go/log"
+)
+
+// EventReceiver implements dbr.TracingEventReceiver for opentracing-go
+type EventReceiver struct{}
```

Comment:
@mccolljr 
```go
var _ dbr.TracingEventReceiver  = EventReceiver{}
```
I think this does not implement dbr.TracingEventReceiver yet because it is missing methods from dbr.EventReceiver.